content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,45 @@
+# Add Output to Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of its output files or directories, which are already in the
+workspace, are missing from `dvc.yaml` (`outs` field). Follow the steps below to
+add existing files or directories as outputs to a stage without re-executing it
+again, which can be expensive/time-consuming, but is unnecessary.
```

Comment:
Here it is. Notice how I clarify the diff between files/dirs written by the stage vs. outputs (in `<abbr>`) tracked by DVC (`outs` field of dvc.yaml). Please apply similar changes to the other 2 docs if you haven't and request my review again to finalize this. Thanks

Reply:
Actually, this is missing `<abbr>` tags around the 2nd instance of "outputs" 🤦. Please add them @imhardikj, thanks.

Reply:
ACTUALLY, you can just commit this (made another small copy edit for extra clarification between written files vs tracked outputs):
```suggestion
There are situations where we have executed a stage (either by writing
`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
that some of the files or directories it writes, which are already in the
workspace, are missing from `dvc.yaml` (`outs` field). Follow the steps below to
add existing files or directories as <abbr>outputs</abbr> to a stage without
re-executing it again, which can be expensive/time-consuming, but is
unnecessary.
```

Reply:
p.s. Not sure whether it should be "it writes" or "it creates" — up to you.

Reply:
used "creates" because every time before execution existing outputs are deleted, and then new ones are created.